run_robot_test.sh

```diff
@@ -1,6 +1,6 @@
 #/bin/bash
 
-TEST_CASE_FILE=tests/Tests/test.robot
```

Comment:
This is a good idea but I think it should moved to a separate PR that also updates the existing tests to support this workflow.  

All of the existing tests in the `tests/Tests/` directory need to be updated to support the same test teardown that would leave the test environment in a state that allows another test to run like closing the browser or adding Test Setup to check if a browser is already open

Reply:
Until the `Tests` directory is refactored, this would automatically run the installation test with the notebook tests

Reply:
Refactored the whole directory in the new commit (0e88e30).
Now all tests pass and install/uninstall are skipped

Reply:
The work for updating to v1.0.10 and the refactor should be two separate PRs. The refactor should be part of a separate discussion on how you want to organize the directory/tests going forward and documenting that for future contributors

Reply:
Split the work in two separate PRs, but we need to decide how to test this one since Jenkins would only run `test.robot`, which is currently broken. See newest comment and/or #34 for more info.